src/adapter/src/metrics.rs

```diff
@@ -59,10 +60,13 @@ impl Metrics {
                 help: "The total number of adapter commands issued of the given type since process start.",
                 var_labels: ["command_type", "status", "application_name"],
             )),
-            storage_usage_collection_time_seconds: registry.register(metric!(
-                name: "mz_storage_usage_collection_time_seconds",
-                help: "The number of seconds the coord spends collecting usage metrics from storage.",
-                buckets: histogram_seconds_buckets(0.000_128, 8.0)
+            storage_usage_collection_last_run_duration: registry.register(metric!(
```

Comment:
```suggestion
            storage_usage_collection_last_run_seconds: registry.register(metric!(
```

Reply:
Recommend changing this to seconds, see https://prometheus.io/docs/practices/naming/.

Reply:
(Or maybe `_duration_seconds`)